security/azure/webapp_tls_version_support/azure_webapp_min_tls_version.pt

```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Security"
 default_frequency "daily"
 info(
-  version: "2.4",
+  version: "2.6",
```

Comment:
Ideally master should have 2.5, I see change change.log 2.5 in the policy it is not updated it. However Juan is updating correct version.